drivers/ethernet/eth_sam_gmac.c

```diff
@@ -515,8 +467,7 @@ static void tx_completed(Gmac *gmac, struct gmac_queue *queue)
 			     queue_list[queue->que_idx]);
 #endif
 
-	__ASSERT(gmac_desc_get_w1(&tx_desc_list->buf[tx_desc_list->tail])
-		 & GMAC_TXW1_USED,
+	__ASSERT(&tx_desc_list->buf[tx_desc_list->tail]->w1 & GMAC_TXW1_USED,
```

Comment:
`tx_desc_list->buf[tx_desc_list->tail].w1`

Reply:
Good catch, that's fixed now.